tests/resources/OVA-Util.robot

```diff
@@ -42,27 +42,54 @@ ${tls_not_disabled}  False
 *** Keywords ***
 # Requires vc credential for govc
 Deploy VIC-OVA To Test Server
-    [Arguments]  ${dhcp}=False  ${build}=False  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL}  ${datastore}=%{TEST_DATASTORE}  ${cluster}=%{TEST_RESOURCE}  ${datacenter}=%{TEST_DATACENTER}
+    [Arguments]  ${dhcp}=True  ${protocol}=http  ${build}=False  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL_ARRAY}  ${datastore}=%{TEST_DATASTORE}  ${cluster}=%{TEST_RESOURCE}  ${datacenter}=%{TEST_DATACENTER}
     Run Keyword if  ${build}  Build Unified OVA
     ${rev}=  Run  git rev-parse --short HEAD
-    Set Test Variable  ${ova_path}  bin/vic-1.1.0-${rev}.ova
+    Set Global Variable  ${ova_path}  bin/vic-1.1.0-${rev}.ova
```

Comment:
Please put this at the top of the file or its own keyword so it's easy to find to update. The 1.1.0 part is probably already outdated.